conf.py

```diff
@@ -266,6 +266,8 @@
     "doc-cheat-sheet*",
     "CONTRIBUTING.md",
     "README.md",
+    ".github/pull_request_template.md",
+    ".github/",
```

Comment:
Could we possibly just only ignore the _.github_ directory here? Ideally Sphinx would just ignore all the contents of this directory:

```suggestion
    ".github/*",
```

Reply:
Honestly, probably. I wasn't quite sure what the formatting was so I was being overly cautious not to have to add five more commits to make it work. Adding that shouldn't hurt.